packages/ui/utils/sortAssets.ts

```diff
@@ -0,0 +1,15 @@
+import { SupportedAsset } from '@midas-capital/types';
+
+import { MarketData } from '@ui/hooks/useFusePoolData';
+
+export const sortAssets = (assets: MarketData[]) => {
+  return assets.sort((a, b) => {
+    return b.underlyingSymbol.toLowerCase() > a.underlyingSymbol.toLowerCase() ? -1 : 1;
```

Comment:
does this work?
We may better use `localeCompare` as often recommened for string sorting, afaik.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare